bcda/utils/common.go

```diff
@@ -79,3 +79,25 @@ func CloseFileAndLogError(f *os.File) {
 		logrus.Error(ferr)
 	}
 }
+
+// This is function that takes a string slice and removes any duplicates
```

Comment:
Can we update this so we start with the declared name? So: `Dedupe takes a string...`

Reference: https://golang.org/doc/effective_go.html#commentary